### PR TITLE
Use some templater for emails make mail util

### PR DESCRIPTION
**Pull request by ko-yaroslav**

### CODE REVIEW

File: server/utils/emailer/nodeEmailer.js [hunk 1]
```diff
@@ -15,19 +30,24 @@ const transport = nodemailer.createTransport({
...
+async function sendEmail(to, subject, templater, context) {
+
   const message = {
     from: smtpEmailFrom, // Sender address
-    to: email, // List of recipients
+    to, // List of recipients
     subject, // Subject line
-    text: content, // Plain text body
+    template: templater || 'welcome',
+    context: {...context}
```
Comment (yaroslavfedyshyn) [inline server/utils/emailer/nodeEmailer.js]: @ko-yaroslav can't it be like this: context, ?

File: server/utils/emailer/views/email/forgotPassword.hbs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <title>Worship Portal</title>
+    <meta name="HandheldFriendly" content="True"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, viewport-fit=cover"/>
+</head>
+<body class="{{template}}">
+<div class="container">
+    {{> header}}
+    <hr/>
+    {{{body}}}
```
Comment (yaroslavfedyshyn) [inline server/utils/emailer/views/email/forgotPassword.hbs]: @ko-yaroslav  what is body doing?

File: server/utils/emailer/nodeEmailer.js (hunk 1 above)
Comment (ko-yaroslav) [reply server/utils/emailer/nodeEmailer.js]: Yes, you`re right. Done.

File: server/utils/emailer/views/email/forgotPassword.hbs (hunk 2 above)
Comment (ko-yaroslav) [reply server/utils/emailer/views/email/forgotPassword.hbs]: Yet nothing. I've cleaned it.

**Pull request merged**